test/dash/mpd_utils_unit.js

```diff
@@ -691,8 +674,8 @@ describe('MpdUtils', () => {
       /** @type {!shaka.util.PublicPromise} */
       const continuePromise = fakeNetEngine.delayNextRequest();
 
-      const xml = parser.parseFromString(baseXMLString, 'text/xml')
-          .documentElement;
+      const xml = /** @type {shaka.extern.xml.Node} */ (
+        shaka.util.TXml.parseXmlString(baseXMLString));
```

Comment:
nit: add two spaces in the indentation

Reply:
The linter reverts to this spacing unfortunately.